Packages/com.chisel.editor/Chisel/Editor/Components/Generators/CSGBox/Editor/CSGBoxEditor.cs

```diff
@@ -12,10 +12,14 @@
 
 namespace Chisel.Editors
 {
+    public sealed class CSGBoxDetails : ChiselGeneratorDetails<CSGBox>
+    {
```

Comment:
Implementation of the meta data for CSGBox, it uses generics to set everything up & uses the default icons. Same with all the other generators